datahub-upgrade/src/main/java/com/linkedin/datahub/upgrade/restorebackup/ClearAspectV2TableStep.java

```diff
@@ -0,0 +1,35 @@
+package com.linkedin.datahub.upgrade.restorebackup;
+
+import com.linkedin.datahub.upgrade.UpgradeContext;
+import com.linkedin.datahub.upgrade.UpgradeStep;
+import com.linkedin.datahub.upgrade.UpgradeStepResult;
+import com.linkedin.datahub.upgrade.impl.DefaultUpgradeStepResult;
+import com.linkedin.metadata.entity.ebean.EbeanAspectV2;
+import io.ebean.EbeanServer;
+import java.util.function.Function;
+
+
+/**
+ * Optional step for removing Aspect V2 table.
+ */
+public class ClearAspectV2TableStep implements UpgradeStep {
+
+  private final EbeanServer _server;
+
+  public ClearAspectV2TableStep(final EbeanServer server) {
+    _server = server;
+  }
+
+  @Override
+  public String id() {
+    return "ClearAspectV2TableStep";
+  }
+
+  @Override
```

Comment:
Just confirming: No skip state required here?

Reply:
For now, I haven't seen any use case for it. The other ones were used by the RestoreIndices task which doesn't require clearing the MySQL table.